test/perf/perf_fast_forward.cc

```diff
@@ -1165,7 +1165,13 @@ static void drop_keyspace_if_exists(cql_test_env& env, sstring name) {
 
 static
 table_config read_config(cql_test_env& env, const sstring& name) {
-    auto msg = env.execute_cql(format("select n_rows, value_size from ks.config where name = '{}'", name)).get0();
+    auto msg = std::invoke([&] {
+        try {
```

Comment:
Why std::invoke(), and not `[] { ... }()`?

Reply:
I think you're not wrong. You'd love LISP.

It's a deviation from the standard idiom, so I need @avikivity 's opinion.

Reply:
I dislike [] () () and like std::invoke.

Reply:
The bike has been shed. Now merge the patch, please.